spec/models/course_enrichment_spec.rb

```diff
@@ -17,6 +17,20 @@
 require 'rails_helper'
 
 describe CourseEnrichment, type: :model do
+  describe 'associations' do
+    it 'belongs to a provider' do
+      expect(subject).to belong_to(:provider)
+                           .with_foreign_key(:provider_code)
+                           .with_primary_key(:provider_code)
+    end
+
+    it 'belongs to a course' do
+      expect(subject).to belong_to(:course)
+                           .with_foreign_key(:ucas_course_code)
```

Comment:
looks odd
ucas_course_code vs course_code

Reply:
This is the way the actual data looks. I know. 🤢